bundles/org.openhab.binding.omnikinverter/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -39,6 +48,60 @@
 		<description>The instantaneous power generation</description>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
+	<channel-type id="powerAC1">
+		<item-type>Number:Power</item-type>
+		<label>Instantaneous Power AC1</label>
+		<description>The instantaneous power generation</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="powerAC2">
+		<item-type>Number:Power</item-type>
+		<label>Instantaneous Power AC2</label>
+		<description>The instantaneous power generation</description>
```

Comment:
Please specialized the description. It is strange to get the same description for different channels.
Check all your new channel descriptions.

Reply:
They're all specialized now.